radio/src/gui/colorlcd/model_inputs.cpp

```diff
@@ -368,6 +368,8 @@ class InputLineButton : public CommonInputOrMixButton
     // if (hasFocus())
     //   textColor = FOCUS_COLOR;
 
+    dc->drawSolidFilledRect(0, 0, width(), height(),
+                            isActive() ? HIGHLIGHT_COLOR : FIELD_BGCOLOR);
```

Comment:
I think this would be better located in `CommonInputOrMixButton::paint`, thus avoiding painting the background twice, and having the same thing implemented twice ;-)

Reply:
You are right, of course.
Please reject this request - I have created another one.

Reply:
Just so you know, you should have to option to close it since you opened it ;) But closing as requested.